worlds/tunic/__init__.py

```diff
@@ -83,6 +83,11 @@ class TunicWorld(World):
     shop_num: int = 1  # need to make it so that you can walk out of shops, but also that they aren't all connected
     er_regions: Dict[str, RegionInfo]  # absolutely needed so outlet regions work
 
+    # so we only loop the multiworld locations once
+    # if these are locations instead of their info, it gives a memory leak error
+    item_link_locations: Dict[int, Dict[str, List[Tuple[int, str]]]] = {}
```

Comment:
To support the change from `if not self.item_link_locations:` to `if self.item_link_locations is not None:` below:
```suggestion
    item_link_locations: Optional[Dict[int, Dict[str, List[Tuple[int, str]]]]] = None
```
`Optional` will also need to be added to the `from typing import` imports at the top of the file.